packages/css/src/components/select/select.scss

```diff
@@ -37,12 +37,9 @@
   @include reset;
 }
 
+.ams-select:invalid,
 .ams-select[aria-invalid="true"] {
```

Comment:
Please add some (initial) documentaion on this API choice and the way people should implement our form fields in this respect.

I would say we can add `storybook/src/docs/forms.docs.mdx` and include it in the ‘Design Guidelines’ group. We’ll tidy up this section soon.

Reply:
Done, I've added it to `packages/css/documentation/coding-conventions.md`